apps/web/components/apps/layouts/AppsLayout.tsx

```diff
@@ -13,14 +13,11 @@ type AppsLayoutProps = {
   actions?: (className?: string) => JSX.Element;
   emptyStore?: boolean;
 } & Omit<ComponentProps<typeof Shell>, "actions">;
-
```

Comment:
Same here. An the rest of these kind of diffs. Our rationale is that we cannot be adapting stylistic preferences for each contributor so we tend to avoid formatting changes were they're not required.